### DISCUSSION TITLE
Understanding the use-case of conduits

### DISCUSSION THREAD

**Discussion by oes-varok** [ProjectOpenSea/seaport#878] [Q&A] [answered]
I have read through the docs, source code and previous discussions and feel I have understood the functionality of conduits; it boils down to giving the user the ability to revoke token permissions on a contract he owns. My question is this:

Most users can’t tell the difference between that and simply granting/revoking permissions off the seaport contract itself, or they don’t care? (Should they? I am not sure I would care?)

Consider this: 
I develop a dapp that uses seaport as a backend contract to handle the nft marketplace, and then get the users to interact with the conduitcontroller to spawn their own conduit and open a channel to seaport. Why bother with this? How does it make any difference to the user?

**Comment by ryanio** [comment, marked as answer]
Users shouldn't be opening their own conduits, only marketplaces. You can also use OpenSea's conduit freely so they don't have to re-ask for approvals

**Reply by oes-varok** [reply]
In that case, should conduits be considered a safeguard against security issues in seaport? Meaning the onus is on the marketplace to block orders when seaport has discovered vulnerabilities?
